package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "1.40.0",
+  "version": "1.41.0",
```

This is a performance fix and doesn't introduce new functionality, so only the patch version should increase.
```suggestion
  "version": "1.40.1",
```